sdk/eventhub/event-hubs/README.md

```diff
@@ -117,69 +116,53 @@ main();
 
 #### Send a batch of events
 
 Use the [createBatch](https://azuresdkdocs.blob.core.windows.net/$web/javascript/azure-event-hubs/5.0.0-preview.6/classes/eventhubproducer.html#createbatch) method to create
-an `EventDataBatch` object which can then be sent using the [send](https://azuresdkdocs.blob.core.windows.net/$web/javascript/azure-event-hubs/5.0.0-preview.6/classes/eventhubproducer.html#send) method.
+an `EventDataBatch` object which can then be sent using the [sendBatch](https://azuresdkdocs.blob.core.windows.net/$web/javascript/azure-event-hubs/5.0.0-preview.6/classes/eventhubproducerclient.html#sendbatch) method.
 Events may be added to the `EventDataBatch` using the [tryAdd](https://azuresdkdocs.blob.core.windows.net/$web/javascript/azure-event-hubs/5.0.0-preview.6/classes/eventdatabatch.html#tryadd)
 method until the maximum batch size limit in bytes has been reached.
 
 ```javascript
 const { EventHubClient } = require("@azure/event-hubs");
 
 async function main() {
-  const client = new EventHubClient("connectionString", "eventHubName");
-  const producer = client.createProducer();
+  const producer = new EventHubProducerClient("connectionString", "eventHubName");
 
   const eventDataBatch = await producer.createBatch();
+  
   let wasAdded = eventDataBatch.tryAdd({ body: "my-event-body" });
   wasAdded = eventDataBatch.tryAdd({ body: "my-event-body-2" });
-  await producer.send(eventDataBatch);
+
+  await producer.sendBatch(eventDataBatch);
 
   await producer.close();
-  await client.close();
 }
 
 main();
 ```
 
 The [Inspect an Event Hub](#inspect-an-event-hub) example shows how to get the list of partition ids should you wish to specify one for a producer.
 
-The `createProducer` method takes an optional parameter of type [EventHubProducerOptions](https://azure.github.io/azure-sdk-for-js/event-hubs/interfaces/eventhubproduceroptions.html) which you can use to specify the retry options and partition id for the send operation.
+The `EventHubProducerClient` constructor takes an optional parameter of type `EventHubClientOptions` which you can use to specify options like number of retries.
 
-The `send` method takes an optional parameter of type [SendOptions](https://azure.github.io/azure-sdk-for-js/event-hubs/interfaces/sendoptions.html) which you can use to specify `abortSignal` to cancel current operation.
-You can also specify `partitionKey` if you did not specify a partition id when creating the producer.
+The `sendBatch` method takes an optional parameter of type `SendBatchOptions` which you can use to specify `abortSignal` to cancel current operation.
 All events that use the same partition key will be sent to the same partition.
```

Comment:
Let's remove this line on partition key, it is not a concept we want to shout from the rooftops at the moment.
```suggestion
```

Reply:
In https://github.com/Azure/azure-sdk-for-js/pull/5964/commits/fa55002b8df0ed60a4a3cb6fa5c258afe30a3d7c, I went ahead and added a section that talks about options at different levels which makes it more clear now.